src/plugins/global-settings/typography/index.js

```diff
@@ -75,11 +95,62 @@ const TYPOGRAPHY_TAGS = [
 	},
 ]
 
+const TYPE_SCALE = [
+	{
+		label: __( 'None', i18n ),
+		value: 'none',
+	},
+	{
+		label: __( 'Custom', i18n ),
+		value: 'custom',
+		disabled: true,
+	},
+	{
+		label: __( '1.067 - Minor Second', i18n ),
```

Comment:
Make the English portion only as translatable:

```suggestion
		// Translators: First %s is a unit, second %s is a name of the unit
		label: sprintf( __( '%s - %s', i18n ), '1.067', __( 'Minor Second', i18n ) ),
```